File: common/src/main/java/de/hype/bbsentials/shared/packets/addonpacket/PlaySoundAddonPacket.java
package de.hype.bbsentials.shared.packets.addonpacket;

import de.hype.bbsentials.environment.addonpacketconfig.AbstractAddonPacket;

/**
 * Plays the specified sound path on the client.
 */
public class PlaySoundAddonPacket extends AbstractAddonPacket {
    public final String namespace;
    public final String path;

    public PlaySoundAddonPacket(String path, String namespace) {
        super(1, 1); //Min and Max supported Version
        this.namespace = namespace;
        this.path = path;
    }
}

File: common/src/main/java/de/hype/bingonet/shared/packets/addonpacket/PlaySoundAddonPacket.kt
package de.hype.bingonet.shared.packets.addonpacket

import de.hype.bingonet.environment.addonpacketconfig.AbstractAddonPacket

/**
 * Plays the specified sound path on the client.
 */
class PlaySoundAddonPacket(val path: String, val namespace: String) : AbstractAddonPacket(1, 1)
